astropy/tests/runner.py

```diff
@@ -48,6 +48,7 @@ def keyword(*args, **kwargs):
         return keyword
 
 
+@deprecated("7.1", alternative="pytest", pending=True)
```

Comment:
When we remove `pending`, we can also bump the version string? Or you rather I put "8.0" now? If the latter, we just have to remember to do it when the time comes.

Reply:
nope, I think the version should be the one it goes in with pending, and when pending is removed then you bump the version.